src/main/java/com/google/sps/servlets/ShadowLoadInterviewsServlet.java

```diff
@@ -0,0 +1,210 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.common.collect.ImmutableList;
+import com.google.common.base.Preconditions;
+import com.google.gson.Gson;
+import com.google.sps.data.Availability;
+import com.google.sps.data.AvailabilityDao;
+import com.google.sps.data.DatastoreAvailabilityDao;
+import com.google.sps.data.DatastorePersonDao;
+import com.google.sps.data.DatastoreScheduledInterviewDao;
+import com.google.sps.data.Job;
+import com.google.sps.data.PersonDao;
+import com.google.sps.data.PossibleInterviewSlot;
+import com.google.sps.data.ScheduledInterview;
+import com.google.sps.data.ScheduledInterviewDao;
+import com.google.sps.data.TimeRange;
+import java.io.IOException;
+import java.io.BufferedReader;
+import java.lang.Integer;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.TextStyle;
+import java.util.EnumSet;
+import java.util.Locale;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.HashSet;
+import java.util.Set;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.ServletContext;
+import javax.servlet.ServletException;
+import javax.servlet.RequestDispatcher;
+import java.util.Optional;
+
+@WebServlet("/shadow-load-interviews")
+public class ShadowLoadInterviewsServlet extends HttpServlet {
+  private ScheduledInterviewDao scheduledInterviewDao;
+  private PersonDao personDao;
+  private Instant currentTime;
+  private final int maxTimezoneOffsetMinutes = 720;
+  private final int maxTimezoneOffsetHours = 12;
+
+  @Override
+  public void init() {
+    init(new DatastoreScheduledInterviewDao(), new DatastorePersonDao(), Instant.now());
+  }
+
+  public void init(
+      ScheduledInterviewDao scheduledInterviewDao, PersonDao personDao, Instant currentTime) {
+    this.scheduledInterviewDao = scheduledInterviewDao;
+    this.personDao = personDao;
+    this.currentTime = currentTime;
+  }
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
long line

Reply:
Formatter inlines it when I try and change it.